src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+function Table() {
```

Comment:
- [ ] Well done on creating a function to fill the page for good practice in REACT to having just one independent function on each file. Please 🙏🏽 divide it into two files one for Table.js and the other for calculator where you will add the corresponding function of the file.
- [ ] This is a REACT file please use it at the top of your document `import REACT from react;` for semantical reasons.